### PR TITLE
updates readme with ubuntu packages

### PR DESCRIPTION
**Pull request by chrisabruce**
- Updates the README to include instructions for installing Ubuntu packages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,15 @@ listening for new block events.
 Once started the application will start syncing the blockchain and
 loading blocks into the attached database.
 
+### Installing Ubuntu Required Packages
+
+If running on Ubuntu, you will need the following packages installed before
+running `make release`:
+
+```bash
+sudo apt install erlang cmake libsodium-dev libssl-dev
```
Comment (madninja) [inline README.md]: @chrisabruce did you now have to add an ESL apt repository to install erlang?

Comment (chrisabruce) [reply README.md]: nope.  Ubuntu 19.04

Comment (madninja) [reply README.md]: but what version of erlang did it actually install?

Comment (chrisabruce) [reply README.md]: 22

**Pull request merged**